exploring.rst

```diff
@@ -0,0 +1,76 @@
+.. _exploring:
+
+Exploring CPython's Internals
+=============================
+
+This is a quick guide for who want to know about CPython's internals,
```

Comment:
"for who want to know" isn't right. Something like "for people interested in learning more about CPython's Internals." seems better. It can then continue with something along the lines of "It provides a summary about the source code structure and contains references to resources providing a more in-depth view."